site/content/docs/include/http-config.en.md

```diff
@@ -31,6 +32,9 @@ http:
 <span class="parent-field">http.healthcheck.</span><a id="http-healthcheck-path" href="#http-healthcheck-path" class="field">`path`</a> <span class="type">String</span>  
 The destination that the health check requests are sent to.
 
+<span class="parent-field">http.healthcheck.</span><a id="http-healthcheck-port" href="#http-healthcheck-port" class="field">`port`</a> <span class="type">Integer</span>
+The port to healthcheck. The default is the `traffic-port` of the target.
```

Comment:
I wonder if folks will be familiar with the `traffic-port` term, maybe we can update it like this. What do you think?
```suggestion
The port to healthcheck. The default is the same as your [`image.port`](./#image-port), the port on which the main container receives traffic. 
```

Reply:
@efekarakus what do you think about this?
```suggestion
The port that the health check requests are sent to. The default is [`image.port`](./#image-port), or the port exposed by [`http.target_container`](./#http-target-container), if set.
```

Reply:
I'm fine with either! 😄